### PR TITLE
Remote access with pgAdmin to Odoo postgre database on Ubuntu

### PR DESCRIPTION
**Pull request by ilyasProgrammer**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,13 +1,16 @@
-Odoo development
+Odoo development
```
Comment (yelizariev) [inline docs/index.rst]: fix it

File: docs/index.rst [hunk 2]
```diff
@@ -1,13 +1,16 @@
-Odoo development
+Odoo development  
 ================
 
 .. toctree::
    :maxdepth: 2
 
+
```
Comment (yelizariev) [inline docs/index.rst]: no need

File: docs/index.rst [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
    install
    ide/index
 
 
+
+
```
Comment (yelizariev) [inline docs/index.rst]: no need extra lines here

**Comment by ilyasProgrammer** [comment]
Corrected

**Pull request merged**